### PR TITLE
Av1e test before upstream.

### PR DESCRIPTION
**Pull request by wenbinc-Bin**

### CODE REVIEW

**Comment by wenbinc-Bin** [comment]
I have merged [https://github.com/intel-media-ci/cartwheel-ffmpeg/pull/192](https://github.com/intel-media-ci/cartwheel-ffmpeg/pull/192) into this PR.

File: libavcodec/qsvenc.c [hunk 1]
```diff
@@ -741,16 +836,22 @@ static int init_video_param(AVCodecContext *avctx, QSVEncContext *q)
...
+            q->param.mfx.QPP = av_clip(quant, 0, 51);
+            q->param.mfx.QPB = av_clip(quant * fabs(avctx->b_quant_factor) + avctx->b_quant_offset, 0, 51);
+        }
         q->old_global_quality = avctx->global_quality;
         q->old_i_quant_factor = avctx->i_quant_factor;
         q->old_i_quant_offset = avctx->i_quant_offset;
         q->old_b_quant_factor = avctx->b_quant_factor;
         q->old_b_quant_offset = avctx->b_quant_offset;
 
+
```
Comment (xhaihao) [inline libavcodec/qsvenc.c]: please avoid unnecessary blank line

Comment (wenbinc-Bin) [reply libavcodec/qsvenc.c]: Done. Thanks

**Review by xhaihao — APPROVED** [review]
LGTM

**Comment by xhaihao** [comment]
see https://patchwork.ffmpeg.org/project/ffmpeg/list/?series=7742

**Pull request closed without merging**